test.js

```diff
@@ -149,3 +149,10 @@ test.cb('parse_file, zip file', function(t) {
 	t.end();
     });
 });
+
+test.cb('parse dependency string', function(t){
+    var deps = desc.parse_dep_string("foo (1.2.3), bar");
```

Comment:
Shouln't this have a `>=`?

Reply:
Yeah that works too. For this test I just did an exact version. Should I change it?

Reply:
OK, updated :)

Reply:
> Yeah that works too. For this test I just did an exact version. Should I change it?

But is that even allowed in R?

Reply:
Not for version requiements no. I use `pkg (1.2.3)` syntax for storing which system dependencies were used to check. I updated the test with a better example.